pyproject.toml

```diff
@@ -85,6 +85,7 @@ dependencies = [
     "refgenconf>=0.12.0",
     "regex",
     "requests",
+    "roc-validator",
```

Comment:
It seems you are using `roc-validator` only for testing. In this case it should go under:

```yaml
[dependency-groups]
dev = [
```
later in this file.

Then you should re-run the commands I mentioned above:

```shell
$ git restore -s upstream/dev lib/galaxy/dependencies/
$ ./lib/galaxy/dependencies/update.sh -p roc-validator
```

To fix the package tests, you just need to add `roc-validator` to `packages/data/test-requirements.txt` .